src/nupic/os/Directory.cpp

```diff
@@ -23,7 +23,9 @@
 /** @file 
 */
 
-#include <unistd.h>
+#if !defined(NTA_OS_WINDOWS)
+  #include <unistd.h>
```

Comment:
This is needed for some functions used, such as `getcwd`. Won't you get compiler errors on Windows since those functions are undefined now?